packages/amplify-provider-awscloudformation/lib/attach-backend.js

```diff
@@ -19,6 +20,13 @@ async function run(context) {
     throw new Error(message);
   }
 
+  const hasPermission = await authDryRun(context, amplifyClient);
+  if (!hasPermission) {
+    const message = 'Permssions to access Amplify service is required.';
+    context.print.error(message);
+    throw new Error(message);
+  }
+
```

Comment:
Is this executed when running `amplify pull`?

Reply:
Yes. 
It is one of the pull workflow when amplify folder is not completely setup for the project.